tests/test_make_whitelist.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from . import call_vulture, v
+assert v  # silence pyflakes
+
+
+@pytest.fixture
+def check_whitelist(v, tmpdir, capsys):
+    def examine(code, expected_out=0, min_confidence=0, sort_by_size=False):
```

Comment:
We don't need to test `min_confidence` and `sort_by_size` here.

Reply:
I'd like to make the tests simpler and faster in the following way (untested):

```
def examine(code, results_before, results_after):
    v.scan(code)
    assert v.get_unused_code() == results_before
    capsys.readouterr()  # Clear captured text (might be unnecessary).
    v.make_whitelist()
    whitelist, _ = capsys.readouterr()
    v.scan(whitelist)
    assert v.get_unused_code() == results_after
```

Reply:
Yes, that works out of the box! ;-)